src/ds4.rs

```diff
@@ -185,13 +245,21 @@ impl<CL: Borrow<Client>> DualShock4Wired<CL> {
 		Ok(())
 	}
 
-	// #[inline(never)]
-	// pub fn update_ex(&mut self, report: &DS4ReportEx) -> Result<(), Error> {
-	// 	if !self.is_attached() {
-	// 		return Err(Error::NotPluggedIn);
-	// 	}
-	// 	unimplemented!()
-	// }
+    #[inline(never)]
+    #[cfg(feature = "unstable_ds4")]
+    pub fn update_ex(&mut self, report: &DS4ReportEx) -> Result<(), Error> {
```

Comment:
This method needs a doc comment, you can just take the comment from the update func above.